library/core/tests/slice.rs

```diff
@@ -2144,3 +2144,12 @@ fn test_slice_run_destructors() {
 
     assert_eq!(x.get(), 1);
 }
+
+#[test]
+fn test_slice_fill_with_uninit() {
+    // This should not UB. See #87891
+    use core::mem::MaybeUninit;
```

Comment:
Please put this `use` at the top of the file.

Reply:
Gotcha, updated